### DISCUSSION TITLE
Error creating bean with name 'taskClient' defined in class path resource  [com/netflix/conductor/client/spring/ConductorClientAutoConfiguration.class]:

### DISCUSSION THREAD

**Discussion by KarthikTMAGL** [Netflix/conductor#3344] [Q&A] [answered]
Hi Team, Can anyone please help on this, tried upgrading spring boot version & conductor client, while deploying to environment getting warning & exception from conductor client jar.

JDK:11, Spring Boot:2.7.5, Conductor client:3.13.2 app is running locally, during deployment into PCF getting below issue. Please help me in understanding what am i missing any config or jar additional config requires

[main] WARN  o.s.b.w.s.c.AnnotationConfigServletWebServerApplicationContext - 
Exception encountered during context initialization - cancelling refresh attempt: 
org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'taskClient' defined in class path resource 
[com/netflix/conductor/client/spring/ConductorClientAutoConfiguration.class]: 
Bean instantiation via factory method failed; nested exception is org.springframework.beans.BeanInstantiationException: 
Failed to instantiate [com.netflix.conductor.client.http.TaskClient]: Factory method 'taskClient' threw exception; 
nested exception is java.lang.NoClassDefFoundError: javax/ws/rs/core/NoContentException

**Comment by manan164** [comment, marked as answer]
Hi @KarthikTMAGL , The [PR](https://github.com/Netflix/conductor/pull/3320) has been raised to fix this issue, https://github.com/Netflix/conductor/discussions/3347#discussioncomment-4159387
